doc/user/content/installation/install-on-aws/appendix-deployment-guidelines.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Appendix: AWS deployment guidelines"
```

Comment:
Copied content from the operation-guidelines.md since we now have enough cloud-specific content to move into cloud-specific pages.